webapp/src/main/webapp/src/app/app.routes.ts

```diff
@@ -1,29 +1,38 @@
-import { HomeComponent } from "./home/home.component";
 import { Routes } from "@angular/router";
 import { UserResolve } from "./user/user.resolve";
 import { GroupsComponent } from "./groups/groups.component";
 import { GroupFilterOptionsResolve } from "./groups/group-filter-options.resolve";
 import { AuthorizeCanActivate } from "./user/authorize.can-activate";
+import { GroupImportComponent } from "./groups/import/group-import.component";
 
 export const routes: Routes = [
   {
     path: '',
-    resolve: { user: UserResolve },
-    data: { permissions: [ 'GROUP_WRITE' ]},
+    resolve: { user: UserResolve, filterOptions: GroupFilterOptionsResolve },
+    data: { permissions: [ 'GROUP_WRITE' ] },
     canActivate: [ AuthorizeCanActivate ],
     children: [
       {
         path: '',
-        pathMatch: 'full',
-        component: HomeComponent
+        pathMatch: 'prefix',
+        redirectTo: 'groups'
```

Comment:
This is different then reporting.  Unfortunately in angular we are not able to have route params on the root route, which I'm using to provide deep linking of the basic filters.  So I'm having to redirect to another route.  In the long run this is probably better anyway in case we should have more functionality other than just groups in the admin app, this allows us to create a home component rather easily to link to the groups page without upsetting all the routing of the app.